Add snap packages

mardy opened the pull request:
This adds a snapcraft recipe for building snap packages (see https://snapcraft.io for more info), which should work in all major Linux distributions.

The included README file gives quick instruction on how to build the snap package.

For the time being, I've uploaded meshlab into the snap store as `meshlab-mardy`, in order to make it clear that that's my personal effort and not something you officially support. But please let me know if you like the idea and would like me to build and publish the package there for you -- I'll do that with pleasure, and I'll add you as an admin so that you can upload packages yourself in the future.

On a side note, as I'm not an expert user of MeshLab, I'd appreciate if you could try out my snap package and tell me if every functionality works as expected. I see for example that the "Polygonal and Quad mesh" submenu in the "Filters" menu is empty, and I wonder if that's somehow my fault.
If you want to try out my package, just type:
```
snap install meshlab-mardy
```
Then you can launch the program from the terminal as `meshlab-mardy.meshlab` (or just launch it via the graphical interface).

cignoni commented:
Thanks! 
I am trying to see if I succeed to test it on my virtualbox ubuntu
 my Linux-fu is rather rusty so i f I succeed it means that it really bullet-proof

mardy commented:
I updated the branch: I implemented your suggestion of using the 2016-12 release, and added a few tweaks to enable the Poisson reconstruction filter according to the links given in issue #93.

cignoni commented:
Great!
I am testing it now on my ubutnt virtual box. it is a 16.04 machine. 
I am not sure about targetting directly the next release of ubuntu (I mean it seems a bit strange that something tagged 2016.12 requires something tagged 2017.04 )
